### DISCUSSION TITLE
Do I need to restart the server after updating the certificate file

### DISCUSSION THREAD

**Discussion by i18nsite** [stalwartlabs/smtp-server#40] [Q&A]
1.
Do I need to restart the server after updating the certificate file ?

2.
I just want a mail delivery server, I don't need to receive mail and full text index, can I skip the configuration of these databases?

3.
Can smtp server send mail through http api? I can't find relevant documents.

4.
I just want the simplest authentication configuration. Is there a simpler way to configure passwords? For example, through a toml configuration file?

5.
Can you give a simple configuration demonstration? How to forward all received emails (catch all for abc.com) to xxx@gmail.com. i have read https://stalw.art/docs/smtp/outbound/routing, but it seems complicated and I don’t understand it.

**Comment by mdecimus** [comment]
Currently you need to restart the server when you install a new certificate. Support for hot reloading certs is going to be added shortly.
